templates/AppConfig.xml.j2

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- {{ ansible_managed }}-->
+<Configuration>
+	<parameters>
+		{% for option,value in syncrify_server_options.items() %}
```

Comment:
das kannst du so nicht machen da syncrify da selber sachen reinschreibt, installier das mal  und klick den installer durch dann siehst was syncrify da noch rein schmeißt

Reply:
ich weiß, das muss man einmal fertig durchklicken dann kann man die relevanten Werte in Ansible übernehmen